README.md

```diff
@@ -86,18 +97,96 @@ determining if something is available. This can be a little slow. If you
 know what you're doing you can speed things up by targeting a single
 region using an environment variable.
 
-```bash
+~~~
 export AWS_REGION=eu-west-1
-```
+~~~
+
+##Getting Started with aws
+
+The aws module allows you to manage AWS using the Puppet DSL. To stand up an instance with AWS, use the `ec2_instance` type. The following code sets up a very basic instance: 
+
+~~~
+ec2_instance { 'instance-name':
+  region          => 'us-west-1',
```

Comment:
Incredibly picky, could you reformat like so. This meets the style guide by having ensure first and having the right amount of whitespace between the property names and the arrows. 

~~~
ec2_instance { 'instance-name':
  ensure        => present,
  region        => 'us-west-1',
  image_id      => 'ami-123456', # you need to select your own AMI
  instance_type => 't1.micro',
}
~~~